### PR TITLE
feat: Source YouTube videos for (issue #6, session 1, April 16th)

### PR DESCRIPTION
**Pull request by raae**

### CODE REVIEW

File: gatsby-node.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  path: `.env.${process.env.NODE_ENV}`,
+});
+
+exports.sourceNodes = async ({actions, createNodeId, createContentDigest}) => {
+  const youtube = google.youtube({
+    version: "v3",
+    auth: process.env.GOOGLE_API_KEY,
+  });
+
+  const response = await youtube.search.list({
```
Comment (raae) [inline gatsby-node.js:12]: This works, no need to use the callback function @PaulieScanlon

Comment (olavea) [reply gatsby-node.js:12]: I am working on this now in my own site:
https://github.com/olavea/snoopy-ears-1.0/issues/77
Yarrr! 🖖 😸 🏴‍☠️

Comment (PaulieScanlon) [reply gatsby-node.js:12]: Yeah it'll work without the callback but you'll still need to handle the error... or maybe not 🤷‍♂️ at some point in dev or prod we might reach the Google 7 day quota and there won't be a response. Without the error callback the process will quit and we won't know why

Comment (raae) [reply gatsby-node.js:12]: I forced an error and it writes out the error in console log. But could of course handle it more gracefully using the Gatsby reporter.

**Comment by PaulieScanlon** [comment]
However you wanna do it it is fine with me boss. Any chance you can merge this in before Tuesday so I can branch off for Fridays PR thingy?

**Pull request merged**